frontend/src/metabase-types/api/search.ts

```diff
@@ -9,14 +9,15 @@ export type EnabledSearchModelType =
   | "collection"
   | "dashboard"
   | "card"
+  | "metric"
```

Comment:
Clicking a metric in search results should open `/metric/:id`, now it opens `/question/:id`